clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -928,7 +928,7 @@ void ContinuationIndenter::addTokenOnCurrentLine(LineState &State, bool DryRun,
   // Note: This doesn't apply to macro expansion lines, which are MACRO( , , )
   // with args as children of the '(' and ',' tokens. It does not make sense to
   // align the commas with the opening paren.
-  if (Style.AlignAfterOpenBracket != FormatStyle::BAS_DontAlign &&
+  if (Style.AlignAfterOpenBracket != false &&
```

Comment:
```suggestion
  if (Style.AlignAfterOpenBracket &&
```
And similar for the others.